platform/web/detect.py

```diff
@@ -170,7 +170,7 @@ def configure(env: "SConsEnvironment"):
     env.AddMethod(create_template_zip, "CreateTemplateZip")
 
     # Closure compiler extern and support for ecmascript specs (const, let, etc).
-    env["ENV"]["EMCC_CLOSURE_ARGS"] = "--language_in ECMASCRIPT_2021"
+    env["ENV"]["EMCC_CLOSURE_ARGS"] = "--language_in UNSTABLE"
```

Comment:
@Faless I changed the "language in" to `UNSTABLE` in order to support static class methods. Otherwise, it doesn't compile.

```
/var/folders/_5/6_c63ny97m554cs3yqq3djxh0000gn/T/emscripten_temp_u8l09o95/godot.web.template_debug.dev.wasm32.nothreads.js:9645:4: ERROR - [JSC_LANGUAGE_FEATURE] This language feature is only supported for UNSTABLE mode or better: Public class fields.
  9645|                 static _samples = new Map();
                        ^

/var/folders/_5/6_c63ny97m554cs3yqq3djxh0000gn/T/emscripten_temp_u8l09o95/godot.web.template_debug.dev.wasm32.nothreads.js:9854:4: ERROR - [JSC_LANGUAGE_FEATURE] This language feature is only supported for UNSTABLE mode or better: Public class fields.
  9854|                 static _sampleNodes = new Map();
                        ^

/var/folders/_5/6_c63ny97m554cs3yqq3djxh0000gn/T/emscripten_temp_u8l09o95/godot.web.template_debug.dev.wasm32.nothreads.js:10064:4: ERROR - [JSC_LANGUAGE_FEATURE] This language feature is only supported for UNSTABLE mode or better: Public class fields.
  10064|                static _buses = [];
                        ^

/var/folders/_5/6_c63ny97m554cs3yqq3djxh0000gn/T/emscripten_temp_u8l09o95/godot.web.template_debug.dev.wasm32.nothreads.js:10066:4: ERROR - [JSC_LANGUAGE_FEATURE] This language feature is only supported for UNSTABLE mode or better: Public class fields.
  10066|                static _busSolo = null;
                        ^
```

Reply:
> `UNSTABLE` doesn't seem something we should target in stable release though.
> 
> Do we really need all those? None of the other libraries needs to use the `static` keyword. Why not use the usual:
> 
> ```
> const Lib = {
>   $Lib: {
>     myProp: 1
>   },
>   godot_js_get_my_prop() {
>     return Lib.myProp;
>   },
> };
> 
> mergeInto(LibraryManager.library, Lib);
> ```

You're right! I reverted back the target.